blogs/eks-canary-deployments-pipeline/microservices/yelb-appserver/specfiles/kubernetes.yml

```diff
@@ -0,0 +1,149 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    mesh: yelb
+    appmesh.k8s.aws/sidecarInjectorWebhook: enabled
+  name: yelb
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: Mesh
+metadata:
+  name: yelb
+spec:
+  namespaceSelector:
+    matchLabels:
+      mesh: yelb
```

Comment:
Sorry, just noticing this -- all the manifests repeat the namespace and mesh. These should be in a separate file that's applied by itself. If someone decides to change this, the odds of missing one instance are pretty good, and the failure modes would be confusing.

Reply:
Added variables to kubernetes_template.yml for namespace, mesh and microservice_name to be substituted by values from deploy.json